### ISSUE TITLE
include/rnd.h missing header guard

### ISSUE THREAD

**Issue by edhartnett** [Unidata/netcdf-c#757]
```
Missing header guard
Source root/include/rnd.h
1
/*



This header file should contain a header guard to prevent multiple inclusion (the macro _RNDUP is checked for, but is not defined).
2
 *      Copyright 1996, University Corporation for Atmospheric Research
3
 *      See netcdf/COPYRIGHT file for copying and redistribution conditions
```

(Same warning for some other files in ncdump: vardara.h, utils.h, nctime0.h, isnan.h, indent.h. I have fixed these as well.)

**Comment by edhartnett** [comment]
I have fixed on my current branch, up as PR #755.

**Comment by edhartnett** [comment]
I took PR #755 down to break its changes into smaller bit-sized pieces.

I have fixed this file on a new branch that will be up for PR shortly. I also removed the unused M_RNDDOWN macro from this header.
